src/python/sdk/examples/event.py

```diff
@@ -1,88 +1,98 @@
 from os import environ
 
-import httpx
-import rel
-import websocket
-import jwt
-
-JWT_SECRET_DEFAULT = "econia_0000000000000000000000000"
-WS_URL_LOCAL_DEFAULT = "ws://0.0.0.0:3001"
-WS_CHANNEL_DEFAULT = "fill_event"
-
-
-def on_message(ws, message):
-    print(message)
-
-
-def on_error(ws, error):
-    print(error)
-
-
-def on_close(ws, close_status_code, close_msg):
-    print("### closed ###")
-
-
-def on_open(ws):
-    print("Opened connection")
-
-
-def get_ws_host() -> str:
-    url = environ.get("WS_URL")
-    if url == None:
-        url_in = input(
-            "Enter the URL of the WebSocket host (enter nothing to default to local OR re-run with WS_URL environment variable)\n"
+import paho.mqtt.client as mqtt
+import json
+
+# This is an example on how to use subscribe to events emitted by the DSS.
```

Comment:
See https://github.com/econia-labs/econia/pull/702#discussion_r1516876928

For ease of onboarding I think this should be default run against a local testnet DSS, so that the user can run the `trade.py` script and see events coming through, rather than having to backfill mainnet and make sense of what is coming through